courses/cunix/ex01/src/my_strlen.c

```diff
@@ -0,0 +1,13 @@
+unsigned int my_strlen(char *str)
+{
```

Comment:
in general brackets should be in same line with func name, or if statement, or for, while etc
```suggestion
unsigned int my_strlen(char *str) {
```